### PR TITLE
DEVDOCS-4913: [update] Webhooks, change store/customer/updated

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4913]

## What changed?
- Change webhook description for store/customer/updated

[DEVDOCS-4913]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4913?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by slsriehl — APPROVED** [review]

**Review by isaacchen-bc — APPROVED** [review]

File: models/webhooks/_all.yml [hunk 1]
```diff
@@ -180,7 +180,7 @@ properties:
     allOf:
       - $ref: ./store_customer_payment_instrument_default_updated.yml
   store/customer/updated:
-    description: Fires when a customer is updated. Does not currently track changes to the customer address.
+    description: Fires when a customer is updated. Does not currently track changes to the customer address. Tracks changes to customer attributes only if the changes are made using the Control Panel.
```
Comment (isaacchen-bc) [inline models/webhooks/_all.yml:183]: FYI: https://github.com/bigcommerce/dev-docs/pull/2042/files#r1289439695

Comment (bc-andreadao) [reply models/webhooks/_all.yml]: 👍 Updated the description based on your feedback

File: models/webhooks/_all.yml [hunk 2]
```diff
@@ -180,7 +180,7 @@ properties:
     allOf:
       - $ref: ./store_customer_payment_instrument_default_updated.yml
   store/customer/updated:
-    description: Fires when a customer is updated. Does not currently track changes to the customer address.
+    description: Fires when a customer is updated. Does not currently track changes to the customer address. <br /><br />Tracks changes to customer attributes only if the changes are made through the Control Panel. This triggers the same event `type` and payload as updating a customer; the payload does not include customer attributes.
```
Comment (bc-traciporter) [inline models/webhooks/_all.yml:183]:
```suggestion
    description: Fires when a customer is updated. It does not currently track changes to the customer address. <br /><br />Tracks changes to customer attributes only when you make changes through the control panel. This change triggers the same event `type` and payload as updating a customer; the payload does not include customer attributes.

```

File: models/webhooks/store_customer_updated.yml [hunk 3]
```diff
@@ -1,7 +1,10 @@
 type: object
 properties:
   store/customer/updated:
-    description: Fires when a customer is updated. Does not currently track changes to the customer address.
+    description: |-
+      Fires when a customer is updated. Does not currently track changes to the customer address.
```
Comment (bc-traciporter) [inline models/webhooks/store_customer_updated.yml:5]:
```suggestion
      Fires when a customer is updated. It does not currently track changes to the customer address.

```

File: models/webhooks/store_customer_updated.yml [hunk 4]
```diff
@@ -1,7 +1,10 @@
 type: object
 properties:
   store/customer/updated:
-    description: Fires when a customer is updated. Does not currently track changes to the customer address.
+    description: |-
+      Fires when a customer is updated. Does not currently track changes to the customer address.
+
+      Tracks changes to customer attributes only if the changes are made through the Control Panel. This triggers the same event `type` and payload as updating a customer; the payload does not include customer attributes.
```
Comment (bc-traciporter) [inline models/webhooks/store_customer_updated.yml:7]:
```suggestion
      Tracks changes to customer attributes only when you make changes through the control panel. This change triggers the same event `type` and payload as updating a customer; the payload does not include customer attributes.

```

**Review by bc-traciporter — COMMENTED** [review]
I left a few suggestions

**Review by bc-traciporter — COMMENTED** [review]
I made an update to my suggestion.

**Review by bc-traciporter — APPROVED** [review]

**Pull request merged**